lib/vdsm/tool/config_lvm_filter.py

```diff
@@ -163,74 +167,65 @@ def parse_args(args):
 
 
 def _print_mounts(mounts):
-    print("Found these mounted logical volumes on this host:")
-    print()
+    log.info("Found these mounted logical volumes on this host:\n")
 
     for mnt in mounts:
-        print("  logical volume: ", mnt.lv)
-        print("  mountpoint:     ", mnt.mountpoint)
-        print("  devices:        ", ", ".join(mnt.devices))
-        print()
+        log.info("  logical volume: %s", mnt.lv)
+        log.info("  mountpoint:     %s", mnt.mountpoint)
+        log.info("  devices:        %s\n", ", ".join(mnt.devices))
 
 
 def _print_recommended_filter(wanted_filter):
-    print("This is the recommended LVM filter for this host:")
+    log.info("This is the recommended LVM filter for this host:\n")
     print()
     print("  " + lvmfilter.format_option(wanted_filter))
     print()
-    print("""\
-This filter allows LVM to access the local devices used by the
-hypervisor, but not shared storage owned by Vdsm. If you add a new
-device to the volume group, you will need to edit the filter manually.
-    """)
+    log.info("This filter allows LVM to access the local devices used by the")
+    log.info("hypervisor, but not shared storage owned by Vdsm. "
+             "If you add a new")
+    log.info("device to the volume group, "
+             "you will need to edit the filter manually.")
```

Comment:
Why to split it (here and elsewhere) to multiple logging messages?  For visual effects?

Reply:
Multiline strings with logging look pretty bad, and it is consensus to try to avoid them. They look misaligned and makes it harder to parse.

In this example, if we keep it in a single string, it would look like:

```
INFO    | This filter allows LVM to access the local devices used by the
hypervisor, but not shared storage owned by Vdsm. If you add a new
device to the volume group, you will need to edit the filter manually.
```